spec/models/embedded_ansible_worker_spec.rb

```diff
@@ -0,0 +1,143 @@
+describe EmbeddedAnsibleWorker do
+  subject { FactoryGirl.create(:embedded_ansible_worker) }
+
+  context "ObjectManagement concern" do
+    let(:provider)       { FactoryGirl.create(:provider_embedded_ansible) }
+    let(:api_connection) do
+      conn = double("AnsibleAPIConnection")
```

Comment:
Actually, now that I read further...
```ruby
let(:tower_api)      { double("TowerAPI", :organizations => org_collection, .....) }
let(:org_collection)       { double("AnsibleOrgCollection", :all => [org_resource]) }
```

Then you can get rid of the `before` block with all of the `allow`s.